pages/common/bmaptool.md

```diff
@@ -0,0 +1,15 @@
+# bmaptool
+
+> Create or Copy blockmaps intelligently (and therefore faster than `cp` or `dd`).
+
+- Create a blockmap into sdb1.
+
+`bmaptool create -o {{/dev/sdb1}}`
+
+- Copy an image file into sdb1.
+
+`bmaptool copy --nobmap {{source.img}} {{/dev/sdv1}}`
+
+- Copy a compressed image file into sdb1.
+
+`bmaptool copy --nobmap {{source.img.gz}} {{/dev/sdb1}}`
```

Comment:
Both of the two `copy` examples here have `--nobmap` specified, which would be somewhat defeating the purpose of the tool. Since you already provided an example for generating the bmap file, how about defaulting to specify it, and optionally add `--nobmap` as another example?